js-envs-test-kit/src/main/scala/org/scalajs/jsenv/test/RunTests.scala

```diff
@@ -142,6 +146,21 @@ private[test] class RunTests(config: JSEnvSuiteConfig, withCom: Boolean) {
     }
   }
 
+  @Test
+  def defaultFilesystem: Unit = {
+    // Tests that a JSEnv works with files from the default filesystem.
+
+    val tmpFile = File.createTempFile("sjs-run-test-defaultfile", ".js")
+    tmpFile.deleteOnExit()
```

Comment:
Could we be more active in deleting this file, for example with a try/finally?

Reply:
Of course. Good point.